main.go

```diff
@@ -172,6 +194,7 @@ func addUserHandler(w http.ResponseWriter, r *http.Request, db *sql.DB) {
 	}
 
 	query := "INSERT INTO users (user_name, user_email, user_bdate) VALUES ($1, $2, $3) RETURNING user_id"
+	fmt.Printf(query, user.Name, user.Email, user.BirthDate)
```

Comment:
Please start using the logger for those kind of activity. This should fit in the DEBUG log level.